lxd-benchmark/benchmark/benchmark.go

```diff
@@ -225,7 +225,7 @@ func ensureImage(c lxd.ContainerServer, image string) (string, error) {
 	var fingerprint string
 
 	if strings.Contains(image, ":") {
-		defaultConfig := config.DefaultConfig
+		defaultConfig := config.NewConfig("", true)
```

Comment:
Does that mean that config.DefaultConfig is no longer going to be working?

If so, that's an API regression in the client library and therefore not acceptable. If NewConfig() is needed to get the new feature but backward compatibility is kept with DefaultConfig, then that's fine.

Reply:
This should be ok. The reason for `NewConfig()` is to add the cookiejar, which is only needed if you enable macaroon auth.

Ideally, though, we should deprecate use of `DefaultConfig` and always go through NewConfig(), in case we need to add logic.